### PR TITLE
93 fix home page on medium and smaller screens

### LINKED ISSUE #93: Fix Home page on medium and smaller screens
## Description

section are getting overlaying each other

## Steps to Reproduce

**Expected behavior:** [What you expected to happen]

**Actual behavior:** [What actually happened]

### PR DESCRIPTION
**Pull request by OmarBassatni97**

### CODE REVIEW

**Review by ayubabdullah — APPROVED** [review]
Good work omer well done

I just have one suggestion can you change the hover color of partner logos to corresponding logo color, like hover color green for Spotify and so on 

Other than that all things work great know

**Comment by OmarBassatni97** [comment]
> Good work omer well done
> 
> I just have one suggestion can you change the hover color of partner logos to corresponding logo color, like hover color green for Spotify and so on
> 
> Other than that all things work great know

Thank you for your review, but this issue is only for fixing the responsiveness of the home page.
You need to create a new issue for what you are asking

**Comment by ayubabdullah** [comment]
> > Good work omer well done
> > I just have one suggestion can you change the hover color of partner logos to corresponding logo color, like hover color green for Spotify and so on
> > Other than that all things work great know
> 
> Thank you for your review, but this issue is only for fixing the responsiveness of the home page. You need to create a new issue for what you are asking

You fixed the hover color for the cards that's why I asked

I will do after this one is approve

**Comment by sarahaboudalal** [comment]
> > > Good work omer well done
> > > I just have one suggestion can you change the hover color of partner logos to corresponding logo color, like hover color green for Spotify and so on
> > > Other than that all things work great know
> > 
> > 
> > Thank you for your review, but this issue is only for fixing the responsiveness of the home page. You need to create a new issue for what you are asking
> 
> You fixed the hover color for the cards that's why I asked
> 
> I will do after this one is approve

It's easy for Spotify yes but when it comes to logos with multiple colors like google it's gonna put u in so much pain (just saying) unless we change the logo SVG when hovering (which is also painful I think). is it worth it?

**Review by sarahaboudalal — APPROVED** [review]
LGTM Omar, well done!

File: src/components/FAQ/index.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ function FAQ() {
   );
 
   return (
-    <section className="bg-background py-20 mx-auto h-screen">
+    <section className="bg-background py-20 mx-auto min-h-screen">
```
Comment (MARWAHAKMI) [inline src/components/FAQ/index.jsx:21]: it will break on the big screen 
image
see this. 
I suggest removing h-screen it will resolve this issue

**Review by MARWAHAKMI — COMMENTED** [review]
Thank you, Omar, I added a small comment for FAQ 
and also for NewsLetter comp. you need to remove top-margin 
see attached image
image

**Pull request merged**